tidb-lightning/import-into-vs-tidb-lightning.md

```diff
@@ -0,0 +1,138 @@
+- 10 TiB of data (565 million rows, single row size 19 KiB)
+
+AWS:
+
+- Importing 40 TiB of data, 10 TiDB (16C32G) 20 TiKV nodes (16C27G), the average single TiDB node import speed is approximately 222 GiB/h.
+- Importing 10 TiB of data, 5 TiDB (16C32G) 10 TiKV (16C27G), the average single TiDB node import speed is approximately 307 GiB/h.
+
+### High availability
+
+#### `IMPORT INTO`
+
+After a TiDB node failure, tasks on that node are automatically transferred to the remaining TiDB nodes to continue running.
+
+#### TiDB Lightning
+
+After a TiDB Lightning instance node failure, you need to perform manual recovery of tasks on a new node based on previously recorded checkpoints.
+
+### Scalability
+
+#### `IMPORT INTO`
+
+Due to the use of Global Sort, data imported into TiKV does not overlap, resulting in better scalability compared with TiDB Lightning.
+
+#### TiDB Lightning
+
+Due to only supporting local sort, data imported into TiKV might overlap when adding TiDB Lightning instances, resulting in more compaction operations for TiKV and limiting scalability relative to `IMPORT INTO`.
+
+## Functionalities not supported by `IMPORT INTO`
+
+Currently, `IMPORT INTO` still lacks some functionalities and cannot fully replace TiDB Lightning in some scenarios, such as:
+
+- Logical import
+
+    Before importing data with `IMPORT INTO`, the target table must be empty. If you need to import data into a table that already contains data, it is recommended to use methods like [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) or direct insertion. TiDB v8.0 supports [bulk DML](/system-variables.md#tidb_dml_type-new-in-v800) for executing large transactions.
+
+- Conflict data handling
+
+    `IMPORT INTO` currently does not support conflict data handling. You need to define the table structure properly to ensure that the data to be imported does not conflict with primary keys (PK) or unique keys (UK). Otherwise, it might cause task failures.
+
+- Importing data into multiple target tables
+
+    Currently, only one target table is allowed for the `IMPORT INTO` SQL statement. If you want to import data into multiple target tables, you need to submit multiple `IMPORT INTO` SQL statements.
+
+In future versions, these functionalities will be supported by `IMPORT INTO`, along with additional enhancements to its capabilities, such as allowing modification of concurrency during task execution and adjusting throughput for writing to TiKV. This will make it more convenient for you to manage tasks.
+
+## Summary
+
+Compared with TiDB Lightning, `IMPORT INTO` has greatly improved  in terms of deployment, resource utilization, task configuration convenience, ease of invocation and integration, automated distributed task scheduling and management, enhanced import stability based on Global Sort, high availability, and scalability. It is recommended that you use `IMPORT INTO` instead of TiDB Lightning in appropriate scenarios.
```

Comment:
```suggestion
Compared with TiDB Lightning, `IMPORT INTO`can be directly executed on TiDB nodes, supports automated distributed task scheduling and [TiDB Global Sort](/tidb-global-sort.md), and offers significant improvements in deployment, resource utilization, task configuration convenience, ease of invocation and integration, high availability, and scalability. It is recommended that you consider using `IMPORT INTO` instead of TiDB Lightning in appropriate scenarios.
```